book/website/reproducible-research/rdm/rdm-article.md

```diff
@@ -12,13 +12,13 @@ The benefit of a Data Article is that your output will be {ref}`peer reviewed<cm
 It also facilitates recognition for datasets through research assessment procedures that are more traditionally focused on publication output.
 Publishing a data paper will therefore increase the visibility, credibility and usability of the data, as well as giving you credit as a data producer.
 
-You can find out more in our {ref}`subchapter on publishing a data article<cm-dif-articles-data>`.
+You can find out more in our {ref}`subchapter on publishing a data article<cm-dif-articles-data>` or you can read a blogpost on '[What a data paper could do for you](https://ekaroune.github.io/The-Open-Archaeobotanist/2021-01-30-what_a_data_paper_could_do_for_you/)'.
 
 (rr-rdm-article-options)=
 ## Options to publish a Data Article
 
 Below you can find some journals that publish data articles or papers.
-The costs information was collected in February 2022. 
+The costs information was collected in July 2023. 
```

Comment:
It might be worth adding something here about what the cost structure is, and the way it works? I remember this being huge news to me when I first learned about it, and making that explicit would probably be really valuable for a researcher

Reply:
I think that may have to go in a separate chapter about APCs!